src/lib/node/message_handler.ts

```diff
@@ -155,6 +156,18 @@ export class NodeMessageHandler {
           }
         );
         return { summary };
+      case NodeCommand.getValue:
+        value = node.getValue<any>(message.valueId);
+        return { value };
+      case NodeCommand.getEndpointCount:
+        const count = node.getEndpointCount();
+        return { count };
+      case NodeCommand.isControllerNode:
+        const isControllerNode = node.isControllerNode();
```

Comment:
Are these things we can just put in the initial state dump of the node ?

Reply:
Yes but these are methods. Not totally clear why because I don't see how they can change

Reply:
Let’s check why they are methods and if nothing weird, add to initial state dump.

Reply:
from Al: isControllerNode could also be a property (maybe it should... v9 is around the corner 🙂 ). getEndpointCount is  dynamic, because the endpoint count could theoretically change during usage 

So should we keep getEndpointCount as a command? I will remove isControllerNode as a command and just add it to the state dump

Reply:
Sounds good.